docsets/NLTK/docset.json

```diff
@@ -1,14 +1,24 @@
 {
     "name": "NLTK",
-    "version": "3.0.0",
+    "version": "3.8.1",
     "archive": "NLTK.tgz",
     "author": {
-        "name": "Aziz Alto",
-        "link": "https://github.com/iamaziz"
+        "name": "Aziz Alto, Xavier Yang",
+        "link": "https://github.com/iamaziz, https://github.com/ivaquero"
     },
     "aliases": [
-    	"Python",
-        "NLTK", 
+        "Python",
+        "NLTK",
         "Natural Language ToolKit"
-        ]
-}
\ No newline at end of file
+    ],
+    "specific_versions": [
+        {
+            "version": "3.8.1",
+            "archive": "versions/3.8.1/NLTK.tgz"
+        },
+        {
+            "version": "3.0.0",
+            "archive": "versions/3.0.0/NLTK.tgz"
```

Comment:
Moving the `txt` file does nothing. You need to add this actual archive, or remove the specific version.